lib/lab_manager/provider/vsphere.rb

```diff
@@ -94,7 +103,7 @@ def create_vm(opts = {})
 
       VSphere.with_connection do |vs|
         dest_folder = opts[:dest_folder]
-        vm_name = opts[:name] || 'lm_' + SecureRandom.hex(8)
+        vm_name = opts[:name] || "#{compute.image}-#{SecureRandom.hex(8)}"
```

Comment:
OK :) An usable compromise could be:
<template>-YYYYMMDD-<guid>

This will get us both idea about creation date, while preserving the guid ensured uniquenes, both achieved with reasonable name length.